### ISSUE TITLE
NotFound: No such file or directory (os error 2): writefile '/x/webui@2.4.1/src/cache/webui-macos-clang-arm64.zip'

### ISSUE THREAD

**Issue by redradist** [webui-dev/deno-webui#28]
Got the following error during testing on MacOS M1:
```console
error: Uncaught (in promise) NotFound: No such file or directory (os error 2): writefile '/x/webui@2.4.1/src/cache/webui-macos-clang-arm64.zip'
  await Deno.writeFile(dest, fileData);
  ^
    at async Object.writeFile (ext:deno_fs/30_fs.js:809:7)
    at async downloadFile (https://deno.land/x/webui@2.4.1/src/utils.ts:18:3)
    at async downloadCoreLibrary (https://deno.land/x/webui@2.4.1/src/utils.ts:125:3)
    at async getLibName (https://deno.land/x/webui@2.4.1/deps.ts:83:5)
    at async https://deno.land/x/webui@2.4.1/deps.ts:88:24

Process finished with exit code 1
```

**Comment by AlbertShown** [comment]
The latest commit did not fix it ether. Deno has no access to `unzip` command under Snap in Linux. And yes, `/x/` should be `./x/`... however, auto-download at runtime does not work for now. Only `bootstrap` is working.

**Comment by AlbertShown** [comment]
Fixed in [v2.4.3](https://github.com/webui-dev/deno-webui/releases/tag/2.4.3)

**Comment by redradist** [comment]
> Fixed in [v2.4.3](https://github.com/webui-dev/deno-webui/releases/tag/2.4.3)

@AlbertShown Yes, now it is working on MacOS !! Thanks !!
